netty/src/main/java/io/grpc/netty/SendResponseHeadersCommand.java

```diff
@@ -53,13 +68,13 @@ public boolean equals(Object that) {
     SendResponseHeadersCommand thatCmd = (SendResponseHeadersCommand) that;
     return thatCmd.stream.equals(stream)
         && thatCmd.headers.equals(headers)
-        && thatCmd.endOfStream == endOfStream;
+        && thatCmd.status.equals(status);
   }
 
   @Override
   public String toString() {
     return getClass().getSimpleName() + "(stream=" + stream.id() + ", headers=" + headers
-        + ", endOfStream=" + endOfStream + ")";
+        + ", status=" + status + ")";
```

Comment:
for line 81:    If equals changes, the hashcode should change too.  I think status is immutable so it shouldn't change.

Reply:
If we want the hashcode to be better, I think we can include all the fields since we're already considering them for `equals`.